.github/workflows/e2e-tests-breaking.yml

```diff
@@ -26,7 +28,10 @@ jobs:
           node-version: "*"
           cache: "yarn"
       - name: Publish
-        run: ./scripts/integration-tests/publish-local.sh
+        id: publish
+        run: |
+          ./scripts/integration-tests/publish-local.sh
+          if [ -e "/tmp/babel_skip_e2e" ]; then echo "skip_e2e=true" >> $GITHUB_OUTPUT; fi
```

Comment:
https://github.com/babel/babel/blob/1203f1858d47254030ec9be21dcc93462fda4a56/scripts/integration-tests/publish-local.sh#L29-L31
This may require copy-pasting multiple lines.

Reply:
Wouldn't it just be like this?
```suggestion
          if [ $? -eq 10 ]; then echo "skip_e2e=true" >> $GITHUB_OUTPUT; fi
```

Reply:
It doesn't seem to work, I manually call `exit 0` in `.sh` to prevent the error from being caught by the github action.